includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -304,6 +303,12 @@ public function wc_setup_store_setup() {
 			$state = '*';
 		}
 
+		$locale_info = include( WC()->plugin_path() . '/i18n/locale-info.php' );
+		function get_currency_code( $country ) {
+			return $country[ 'currency_code' ];
+		}
+		$currency_by_country = array_map( 'get_currency_code', $locale_info );
```

Comment:
You should be able to avoid defining a new callback here by using [`wp_list_pluck()`](https://developer.wordpress.org/reference/functions/wp_list_pluck/).